2021-11-15 17:57:25,057 INFO    MainThread:46300 [wandb_setup.py:_flush():71] setting env: {}
2021-11-15 17:57:25,057 INFO    MainThread:46300 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-15 17:57:25,057 INFO    MainThread:46300 [wandb_init.py:_log_setup():357] Logging user logs to D:\_jupyter\kaggle\notebooks\wandb\run-20211115_175725-34olj6u3\logs\debug.log
2021-11-15 17:57:25,057 INFO    MainThread:46300 [wandb_init.py:_log_setup():358] Logging internal logs to D:\_jupyter\kaggle\notebooks\wandb\run-20211115_175725-34olj6u3\logs\debug-internal.log
2021-11-15 17:57:25,058 INFO    MainThread:46300 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000016637E2B490>
2021-11-15 17:57:25,062 INFO    MainThread:46300 [wandb_init.py:init():390] calling init triggers
2021-11-15 17:57:25,063 INFO    MainThread:46300 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-15 17:57:25,063 INFO    MainThread:46300 [wandb_init.py:init():435] starting backend
2021-11-15 17:57:25,063 INFO    MainThread:46300 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-11-15 17:57:25,066 INFO    MainThread:46300 [backend.py:ensure_launched():199] starting backend process...
2021-11-15 17:57:25,156 INFO    MainThread:46300 [backend.py:ensure_launched():204] started backend process with pid: 42908
2021-11-15 17:57:25,156 INFO    MainThread:46300 [wandb_init.py:init():444] backend started and connected
2021-11-15 17:57:25,167 INFO    MainThread:46300 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-15 17:57:25,169 INFO    MainThread:46300 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-15 17:57:25,169 INFO    MainThread:46300 [wandb_init.py:init():503] updated telemetry
2021-11-15 17:57:25,189 INFO    MainThread:46300 [wandb_init.py:init():533] communicating current version
2021-11-15 17:57:26,672 INFO    MainThread:46300 [wandb_init.py:init():538] got version response 
2021-11-15 17:57:26,673 INFO    MainThread:46300 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-15 17:57:26,912 INFO    MainThread:46300 [wandb_init.py:init():576] starting run threads in backend
2021-11-15 17:57:31,926 INFO    MainThread:46300 [wandb_run.py:_console_start():1693] atexit reg
2021-11-15 17:57:31,927 INFO    MainThread:46300 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-15 17:57:31,927 INFO    MainThread:46300 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-15 17:57:31,929 INFO    MainThread:46300 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-15 17:57:31,929 INFO    MainThread:46300 [wandb_init.py:init():603] run started, returning control to user process
2021-11-15 17:57:31,929 INFO    MainThread:46300 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 17:57:31,929 INFO    MainThread:46300 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 17:57:31,956 INFO    MainThread:46300 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 17:57:39,905 INFO    MainThread:46300 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 17:57:39,905 INFO    MainThread:46300 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 17:57:39,925 INFO    MainThread:46300 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 17:58:04,887 INFO    MainThread:46300 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 17:58:04,887 INFO    MainThread:46300 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 17:58:04,907 INFO    MainThread:46300 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 17:58:05,882 INFO    MainThread:46300 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 17:58:05,882 INFO    MainThread:46300 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 17:58:32,592 INFO    MainThread:46300 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-15 17:58:32,593 INFO    MainThread:46300 [wandb_run.py:_restore():1635] restore
2021-11-15 17:58:32,824 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2021-11-15 17:58:33,046 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2021-11-15 17:58:33,417 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 16172
}

2021-11-15 17:58:33,529 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 16199
}

2021-11-15 17:58:33,636 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 16199
}

2021-11-15 17:58:33,741 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}

2021-11-15 17:58:33,846 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}

2021-11-15 17:58:33,951 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}

2021-11-15 17:58:34,056 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}

2021-11-15 17:58:34,161 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}

2021-11-15 17:58:34,603 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}

2021-11-15 17:58:35,111 INFO    MainThread:46300 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16199
  total_bytes: 16199
}
local_info {
}

2021-11-15 17:58:36,224 INFO    MainThread:46300 [wandb_run.py:_append_files():2061] logging synced files
